pydis_site/templates/events/pages/pyweek/_index.html

```diff
@@ -0,0 +1,30 @@
+{% extends "events/base_sidebar.html" %}
+
+{% block breadcrumb %}
+    <li><a href="{% url "events:index" %}">Events</a></li>
+    <li class="is-active"><a href="#">PyWeek</a></li>
+{% endblock %}
+
+{% block title %}PyWeek{% endblock %}
+
+{% block event_content %}
+<p>
+    For the past 15 years, <a href="https://pyweek.org" target="_blank" rel="noopener">PyWeek</a> has been running a bi-annual game jam for the
+    Python language. As of 2020, we are excited to say we are officially partnered with PyWeek to co-run these
```

Comment:
Hmm, I'm not sure if this is necessary from a grammatical perspective, but I feel it reads more smoothly this way:
```suggestion
    Python language. As of 2020, we are excited to say that we are officially partnered with PyWeek to co-run these
```
What do you think?

Reply:
This was copied from the current description on our events page, but I agree! It's a good suggestion -- it sounds better.

Reply:
ece9c315

Also fixed another instance of this content on the annual events gallery.